documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -26,6 +32,18 @@ alias samm='java -jar /location/to/samm-cli-{esmf-sdk-version}.jar'
 ----
 ====
 
+*For the Windows native binary:*
+. *Download*: download above or visit our github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download executable file for Windows.
+. *Extract*: extract it to a location of your choice.
+. *Open Command Prompt*: Launch the Command Prompt and navigate to the directory (samm-cli-windows) containing our application.
+. *Run*: In the command prompt, execute the command ``./samm.exe` to start the application.
+
+*For the Linux native binary:*
+. *Download*: download above or visit our github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download executable file for Linux.
+. *Extract*: Unpack the archive to your preferred directory.
+. *Open Terminal*: Open the Terminal and navigate to the directory with the application.
+. *Execute*: Run the application using the command `./samm`.
+
```

Comment:
. *Input*: Make sure to read the below documentation and provide model files in the correct directory structure

((perhaps link to https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html#models-directory-structure from the same page. This is a bit of duplication, but I fear people will at that point say "got it, don't need the documentation" and then get invalid models, particularly if it's only one and not in the right folder structure. This comment should be relaxed once the SAMM CLI has been made more convenient, see https://github.com/eclipse-esmf/esmf-sdk/issues/495 ))